recipes/mppp/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+import os
+import functools
+
+required_conan_version = ">=1.43.0"
+
+class MpppConan(ConanFile):
+    name = "mppp"
+    description = "Multiprecision for modern C++ Topics"
+    license = "MPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bluescarni/mppp/"
+    topics = ("multiprecision", "gmp", "math-bignum", "computer-algebra")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_mpfr": [True, False],
+        "with_arb": [True, False],
+        "with_mpc": [True, False],
+        "with_quadmath": [True, False],
+        "with_boost": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_mpfr": False,
+        "with_arb": False,
+        "with_mpc": False,
+        "with_quadmath": False,
+        "with_boost": False,
+    }
+    generators = "cmake", "cmake_find_package_multi"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("gmp/6.2.1")
+        if self.options.with_mpfr == True:
+            self.requires("mpfr/4.1.0")
+        if self.options.with_mpc == True:
+            self.requires("mpc/1.2.0")
+        if self.options.with_boost == True:
```

Comment:
```suggestion
        if self.options.with_mpfr:
            self.requires("mpfr/4.1.0")
        if self.options.with_mpc:
            self.requires("mpc/1.2.0")
        if self.options.with_boost:
```
Dont need to check if is True, because python does it implicit ;)